src/main.c

```diff
@@ -127,16 +127,15 @@ main (int argc, char *argv[])
 
   }
   while (next_option != -1);
-
-  if (getenv ("HOME") == NULL)
+  
+  char *HOMEDIR = getenv ("HOME");
+  
+  if (HOMEDIR == NULL)
```

Comment:
opposite, but same effect. If HOMEDIR != NULL the else will be skipped over. If HOMEDIR == NULL the message will get printed.